core/src/main/res/values/strings.xml

```diff
@@ -148,6 +170,13 @@
     <string name="new_episode_notification_group_text">Your subscriptions have new episodes.</string>
 
     <!-- Actions on feeds -->
+    <string name="mark_all_label">Mark all as...</string>
+    <string-array name="mark_all_array">
+        <item>@string/played_label</item>
+        <item>@string/remove_all_new_flags_label</item>
+    </string-array>
+    <string name="mark_all_confirmation_msg">Please confirm that you want to mark all episodes as %s.</string>
+    <string name="removed_all_msg">Marked all \"%s\" flags</string>
```

Comment:
Hmm, this also sounds like it should be its own PR and discussed individually.

Reply:
Yea this could be broken up, or removed, I don't know why I did this, I think the idea was to easily modify all episodes in a given feed.